src/container/srv_metrics.c

```diff
@@ -29,31 +29,36 @@ ds_cont_metrics_alloc(const char *path, int tgt_id)
 	if (metrics == NULL)
 		return NULL;
 
-	rc = d_tm_add_metric(&metrics->cpm_open_count, D_TM_COUNTER,
-			     "Number of times cont_open has been called",
+	rc = d_tm_add_metric(&metrics->open_total, D_TM_COUNTER,
+			     "Total number of container open operations",
```

Comment:
Question: Since we only increment on success, do we want to be explicit?
```suggestion
			     "Total number of successful container open operations",
```